palm/palm_config.py

```diff
@@ -47,10 +66,6 @@ def validate_branch(self) -> None:
         secho(msg, fg="red")
         raise SystemExit(msg)
 
-    @property
-    def has_config(self) -> bool:
```

Comment:
This property was unused outside of tests, and no longer makes sense since there will always be a global config!